ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -110,7 +125,9 @@ class ConfirmToProceedTransactionScreen extends React.Component<Props, never> {
                 </Col>
                 <Col>
                   <Text bold={true} style={WalletStyles.textRight}>
-                    {`${transaction.amount.toFixed(2)}  €`}
+                    {`${AmountInEuroCentsFromNumber.encode(
```

Comment:
I suggest to do all conversions and computations outside (before) the TSX block, having code inside the TSX props makes the structure of the component less readable